pages/common/afconvert.md

```diff
@@ -0,0 +1,12 @@
+# afconvert
+
+> Convert between AFF and raw file formats.
+> More information: <https://manned.org/afconvert.1>.
```

Comment:
wondering if we should have the `afconvert.1` instead of plain old `afconvert` ?

Reply:
`afconvert` seems fine here. In some cases we would go with `.1` if there is a conflicting command in manned with the same name.

Reply:
Yeah, I think we should use suffices like `.1` just to avoid ambiguity.